pkg/storage/replica_consistency.go

```diff
@@ -126,6 +126,10 @@ func (r *Replica) CheckConsistency(
 		// isn't duplicated except in rare leaseholder change scenarios (and concurrent invocation of
 		// RecomputeStats is allowed because these requests block on one another). Also, we're
 		// essentially paced by the consistency checker so we won't call this too often.
+		f := log.Infof
+		if !results[0].Response.Delta.ContainsEstimates {
+			f = log.Warningf
+		}
 		log.Infof(ctx, "triggering stats recomputation to resolve delta of %+v", results[0].Response.Delta)
```

Comment:
Don't you need to actually use `f`? :)

Although I'm not sure why we wouldn't just use a different error message as well here to make it clearer that the recomputation isn't expected in such cases.